docs/CLI.md

```diff
@@ -38,7 +38,6 @@ Options
   --drive, -d    drive
   --check, -c    validate write
   --robot, -r    parse-able output without interactivity
-  --log, -l      output log file
```

Comment:
Why does the ability for the parent and child to communicate via IPC mean that the ability to write a logfile should be removed? Surely it's the 'robot' option that should be removed, if anything?
Apologies if I'm misunderstanding things! ;)